Sources/SpeziOpenAI/MessagesView.swift

```diff
@@ -43,7 +42,7 @@ public struct MessagesView: View {
             ScrollView {
                 VStack {
                     ForEach(Array(chat.enumerated()), id: \.offset) { _, message in
-                        MessageView(message)
+                        MessageView(message, hideMessagesWithRoles: MessageView.Defaults.hideMessagesWithRoles)
```

Comment:
If you want to enable the functionality to pass the argument down to the message views, you will need to pass the arguments down to the individual view, not the default values:
```suggestion
                        MessageView(message, hideMessagesWithRoles: hideMessagesWithRoles)
```